src/data.js

```diff
@@ -26,10 +26,11 @@ var Client = Entity.configure({
     expires:        Entity.types.Date,
     /**
      * Details object with properties:
-     * - created          // Time when client was created
-     * - lastModified     // Last time client was modified
-     * - lastDateUsed     // Only updated if more than 6 hours out of date
-     * - lastRotated      // Last time accessToken was reset
+     * - created            // Time when client was created
+     * - lastModified       // Last time client was modified
+     * - lastDateUsed       // Only updated if more than 6 hours out of date
+     * - lastRotated        // Last time accessToken was reset
+     * - deleteOnExpiration // If true, can be deleted after expiration
```

Comment:
One could argue that we should make `deleteOnExpiration` a top-level property so we can filter on it later... But we're probably not going to have so many expired clients that quering is going to be a problem...